float64 speed1
uint16  current1
bool    status1
float64 speed2
uint16  current2
bool    status2
